src/src/AppBundle/Controller/DefaultController.php

```diff
@@ -57,7 +57,7 @@ public function indexAction($slug, $_locale, Request $request)
         $em = $this->getDoctrine()->getManager();
 
         $registration = new Registration();
-        $token = md5(time().rand()); //TODO
+        $token = md5(time().rand()); // TODO
         $registration->setConfirmationToken($token);
         $registration->setCode(9); // TODO
```

Comment:
Tieto TODO bude tiez treba dorobit.

Reply:
hej, jedno je uz urobene, preto je tam zmena, len som si uvedomila, ze to nepatri do tohto issue